doc/voms-proxy-fake.1

```diff
@@ -221,7 +221,7 @@ The created certificate will be a self\-signed certificate and have a CA=true bi
 \fB\-extension\fR
 \fIoid[/criticality]value\fR
 .PP
-This option allows to specified additional extensions to be put in the created certificate\&.
+This option allows one to specified additional extensions to be put in the created certificate\&.
```

Comment:
```suggestion
This option allows one to specify additional extensions to be put in the created certificate\&.
```
(multiple times)
I don't like much that "one", but if it pleases the tool, it's fine with me.

Reply:
I have updated the "to specified" → "to specify" in three more places.

Reply:
If you do not like the "one" subject, you can rewrite in passive which does not need a subject:

This option allows additional extensions to be put in the created certificate to be specified.

But the sentence becomes very convoluted. Or rewrite to use a different verb than allow:

This option makes it possible to specify additional extensions to be put in the created certificate.